ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -2909,3 +2909,37 @@ def test_numpy_instance_mod__(
         method_flags=method_flags,
         on_device=on_device,
     )
+
+#ptp    
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="ptp",
+    ret_np_flat=np_frontend_helpers._flatten_frontend_return(
```

Comment:
This helper is not supposed to be used for data generation. I think `dtype_values_axis` is the most suitable helper in your case.